app/helper/rtForm.php

```diff
@@ -243,20 +246,21 @@ private function generate_element_value( $element, $attributes ) {
 				case 'rtNumber': //number
 				case 'rtText' : //text
 				case 'rtButton': //button
+				case 'rtFile': //file
```

Comment:
the inline comment must end with full stop need a space between `comment` and `//`